### DISCUSSION TITLE
如何将中文摘要中”关键词“几个字加粗

### DISCUSSION THREAD

**Discussion by chloenkiwi** [sjtug/SJTUThesis#973] [提问]
如题，latex模板得到的是没有加粗的版本

**Comment by skyzh** [comment]
您使用的字体可能没有粗体字重。可以考虑安装 Adobe 系列字体后使用 `cjk-font=adobe` 文档选项。也可以参考这个教程手动下载 windows 系列字体：https://github.com/sjtug/SJTUThesis/wiki/%E5%9C%A8%E7%BA%BF%E4%BD%BF%E7%94%A8%E8%AF%B4%E6%98%8E
